src/lib/create-config.ts

```diff
@@ -402,6 +406,10 @@ export function createConfig(options: IUnleashOptions): IUnleashConfig {
         DEFAULT_STRATEGY_SEGMENTS_LIMIT,
     );
 
+    const frontendApiOrigins =
```

Comment:
Should we default to `*` if no origins are passed?

Reply:
Not sure, seems safer to disallow everything by default. Could always use a different default for `cloud`.